### ISSUE TITLE
In minibatch.py, why need to change channels?Maybe we can remove ' im = im[:,:,::-1]'?

### ISSUE THREAD

**Issue by dydxdt** [jwyang/faster-rcnn.pytorch#774]
In minibatch.py, I find the following code: 
from scipy.misc import imread

for i in range(num_images):
    #im = cv2.imread(roidb[i]['image'])
    im = imread(roidb[i]['image'])
    if len(im.shape) == 2:
       im = im[:,:,np.newaxis]
       im = np.concatenate((im,im,im), axis=2)
     # flip the channel, since the original one using cv2
     # rgb -> bgr
     im = im[:,:,::-1]
"

Since we don't use cv2, we don't need to change channels,right? Because there's something wrong with colors of the input 'im_data' when I try to visualize them.

**Comment by loolzaaa** [comment]
Author used caffe weights as pretrain. Caffe used openCV and BGR format of image, so I think if you use `scipy` for image open, you need to convert RGB to BGR format.

**NOTE!** In my implementation of [this method](https://github.com/loolzaaa/faster-rcnn-pytorch/blob/536411ff30eeed175130e8fcb37a4c4a9684f49b/lib/dataset/collate.py#L34-L61), i use openCV, and i convert BGR to RGB. **Even if i use caffe weights, network can get 74.5% on PascalVOC 2007 with Resnet101 backbone.** So, i think this is not very important for network.
